### PR TITLE
modified the html and css file implemented the file

### PR DESCRIPTION
**Pull request by richie1988**
created the HTML file and CSS
added the images folder
added the implementation css

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -1,10 +1,5 @@
-/* stylelint-disable csstree/validator */
-
-
 /* stylelint-disable indentation */
```
Comment (shella12) [inline style.css]: - [ ] Please don't disable the linters using inline comments instead I would highly recommend fixing the linter errors manually. Linters can help enforce a specific coding style, so code reviews are about the implemented changes and less about style or code formatting conventions. Moreover the use of linters helps to diagnose and fix technical issues—e.g. code smells—in the code. As a result, fewer defects make their way to production. Linters can help teams achieve a more readable and consistent style, through the enforcement of its rules.

**Review by shella12 — CHANGES_REQUESTED** [review]
Hi @richie1988 , 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- Great use of semantic tags ✅ 
- Clean code ✅   
- Great design implementation ✅ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (richie1988) [reply style.css]: its the Vscode problem try to check properly unless if the linters did not pass im having problem with Vscode tried to do research about it nothing coming

Comment (richie1988) [reply style.css]: its not the Code problem its the  computer try to double check properly

Comment (richie1988) [reply style.css]: maybe its in your code cause the top part of the code its not there

Comment (richie1988) [reply style.css]: there is only one which is Vscode problem which is indentation

**Review by Shisui6 — APPROVED** [review]
Hi @richie1988, 

# Approved 🪄
Your project is complete! There is nothing else to say other than... it's time to merge it 💎
Congratulations! 🎉 

## Highlights 🎯
- Descriptive PR title and summary ✅
- Follows Github flow ✅
- Good commit messages ✅

## Optional suggestions 🪶

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer, and addressing them will really improve your application._

You can also consider the following:

N/A

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**